receiver/hostmetricsreceiver/internal/scraper/processscraper/metadata.yaml

```diff
@@ -103,3 +103,30 @@ metrics:
       value_type: int
       aggregation: cumulative
       monotonic: true
+
+  process.open_file_descriptors:
+    enabled: false
+    description: Number of file descriptors in use by the process.
+    unit: '{file descriptors}'
+    sum:
+      value_type: int
+      aggregation: cumulative
+      monotonic: false
+  
+  process.voluntary_ctx_switches:
+    enabled: false
+    description: Number of times the process has been voluntarily context switched.
+    unit: '{context switches}'
+    sum:
+      value_type: int
+      aggregation: cumulative
+      monotonic: true
+
+  process.involuntary_ctx_switches:
+    enabled: false
+    description: Number of times the process has been involuntarily context switched.
+    unit: '{context switches}'
+    sum:
+      value_type: int
+      aggregation: cumulative
+      monotonic: true
```

Comment:
new line?

Reply:
Added it back, thanks for calling this out.